diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/components/SocialProfiles/style.module.css b/node_modules/gatsby-theme-portfolio-minimal/src/components/SocialProfiles/style.module.css
index 4c2b3a3..45df9a0 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/components/SocialProfiles/style.module.css
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/components/SocialProfiles/style.module.css
@@ -7,7 +7,7 @@
     background-position: right bottom;
     border-radius: var(--border-radius);
     border: 0.125rem solid var(--primary-color);
-    padding: 0.3rem 1.25rem;
+    padding: 0.5rem 1.25rem;
     margin-right: 0.5rem;
     transition: all 0.1s ease-out;
     font-size: 1rem;
diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Projects/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Projects/index.tsx
index 35a9dfb..a1b0147 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/sections/Projects/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/sections/Projects/index.tsx
@@ -13,24 +13,24 @@ export function ProjectsSection(props: PageSection): React.ReactElement {
     const data = response.allProjectsJson.sections[0];
 
     return (
-        <Animation type="fadeIn">
-            <Section anchor={props.sectionId} heading={props.heading}>
-                <Slider additionalClasses={[classes.Projects]}>
-                    {data.projects.map((project, key) => {
-                        return project.visible ? <Project key={key} index={key} data={project} /> : null;
-                    })}
-                </Slider>
-                {data.button !== undefined && data.button.visible !== false && (
-                    <Animation className={classes.MoreProjects} type="fadeIn">
-                        <Button
-                            type={ButtonType.LINK}
-                            externalLink={true}
-                            url={data.button.url}
-                            label={data.button.label}
-                        />
-                    </Animation>
-                )}
-            </Section>
-        </Animation>
+
+        <Section anchor={props.sectionId} heading={props.heading}>
+            <Slider additionalClasses={[classes.Projects]}>
+                {data.projects.map((project, key) => {
+                    return project.visible ? <Project key={key} index={key} data={project} /> : null;
+                })}
+            </Slider>
+            {data.button !== undefined && data.button.visible !== false && (
+                <Animation className={classes.MoreProjects} type="fadeIn">
+                    <Button
+                        type={ButtonType.LINK}
+                        externalLink={true}
+                        url={data.button.url}
+                        label={data.button.label}
+                    />
+                </Animation>
+            )}
+        </Section>
+
     );
 }
